elasticsearch/src/main/scala/com/velocidi/apso/elasticsearch/ElasticsearchBulkInserter.scala

```diff
@@ -85,20 +85,19 @@ class ElasticsearchBulkInserter(
     }
   }
 
-  private[this] def countSuccessfulMsgs(sentBuffer: Iterable[Message], bulkResponse: BulkResponse) = {
+  // returns a list of the failed requests
```

Comment:
Can you use proper scaladoc here (and maybe add a type annotation to the method)?

Reply:
Yes, done in e546c98